tools/agents/doctor.py

```
@@ -0,0 +1,185 @@
+    #    וה-job מדולג בירוק לנצח. בדיוק הבאג של #266 בצורתו הבאה.
+    written = set(re.findall(r"echo\s+\"?(\w+)=", body))
+    written |= set(re.findall(r"^\s{4,}(\w+):\s*\$\{\{\s*steps\.", body, re.M))
+    for m in re.finditer(r"\boutputs\.(\w+)", body):
+        if m.group(1) not in written:
+            findings.append(("if-orphan-output",
+                             "%s: ‏`outputs.%s` שאיש אינו כותב — התנאי "
+                             "תמיד false וה-job מדולג בירוק"
+                             % (name, m.group(1))))
+
+    # 3. ‏workflow בלי שום טריגר בר-הפעלה לא ירוץ לעולם.
+    on_m = re.search(r"^(?:True|on):\s*$", body, re.M)
```

Ruff (0.16.3)

[error] Contains control characters that can permit obfuscated code

(PLE2502)